druid/src/widget/image.rs

```diff
@@ -173,30 +219,10 @@ impl ImageData {
         let image_data = image::open(path).map_err(|e| e)?;
         Ok(ImageData::from_dynamic_image(image_data))
     }
-
-    /// Get the size in pixels of the contained image.
-    fn get_size(&self) -> Size {
-        Size::new(self.x_pixels as f64, self.y_pixels as f64)
-    }
-
-    /// Convert ImageData into Piet draw instructions.
-    fn to_piet(&self, offset_matrix: Affine, ctx: &mut PaintCtx, interpolation: InterpolationMode) {
-        ctx.with_save(|ctx| {
-            ctx.transform(offset_matrix);
-            let size = self.get_size();
-            let im = ctx
-                .make_image(
-                    size.width as usize,
-                    size.height as usize,
-                    &self.pixels,
-                    self.format,
-                )
-                .unwrap();
-            ctx.draw_image(&im, size.to_rect(), interpolation);
-        })
-    }
 }
 
+#[cfg(feature = "image")]
+#[cfg_attr(docsrs, doc(cfg(feature = "image")))]
 fn has_alpha_channel(image: &image::DynamicImage) -> bool {
```

Comment:
This being a private function, it won't need the doc config. Also perhaps this makes sense to move under the `ImageData` impl block as an assocaited function.

Reply:
yeah, just got a little trigger happy with the copy paste without thinking. i thought to put this as an associated fn, but decided against it at first since it wasn't necessary (trying to be more atomic with my git commits). easy enough, though

Reply:
looking at it now, i'm thinking i may just inline this function. it's only used in once place, and doesn't totally make sense as an associated fn (doesn't need `self` or `Self`).

Reply:
Associated functions by definition never need `self`. Those are methods.